wallet/src/cli/account.rs

```diff
@@ -25,10 +25,10 @@ struct TokenDefinition {
     total_supply: u128,
 }
 
-struct TokenHolding {
+pub struct TokenHolding {
     #[allow(unused)]
     account_type: u8,
-    definition_id: AccountId,
+    pub definition_id: AccountId,
```

Comment:
Let's make all pub then? So that we either have a [POD](https://en.wikipedia.org/wiki/Passive_data_structure) with all things public and no incapsulated behavior or object with private fields and methods